detox/src/errors/DetoxConfigErrorComposer.js

```diff
@@ -535,6 +552,22 @@ You have a few options:
     });
   }
 
+  invalidCloudAppUrl( appPath ) {
+    return new DetoxConfigError({
+      message: `Invalid "app" property in the app config.\nExpected a string:.`,
+      debugInfo: this._focusOnAppConfig(appPath, this._ensureProperty('app')),
+      inspectOptions: { depth: 4 },
+    });
+  }
+
+  invalidCloudAppClientUrl( appPath ) {
+    return new DetoxConfigError({
+      message: `Invalid "appClient" property in the app config.\nExpected a string:.`,
+      debugInfo: this._focusOnAppConfig(appPath, this._ensureProperty('appClient')),
+      inspectOptions: { depth: 4 },
+    });
+  }
```

Comment:
@pb2323  Can we reuse recipeUUID validation here ?

Reply:
`recipeUUID` does not have any type of validation specifically. No other existing error messages can be used here.